### DISCUSSION TITLE
Code Journey 编程之旅

### DISCUSSION THREAD

**Discussion by giscus** [Open-BJUT/CodeJourney#2] [Announcements]
# Code Journey 编程之旅

https://open-bjut.github.io/CodeJourney/

**Comment by Nagi-ovo** [comment]
欢迎大家参与本次“编程之旅——零基础学习编程”活动，本人才疏学浅，难免词不达意，还请谅解。
本网站仅做引导，没有课程广告、资源售卖等商业行为。请勿发布任何广告，包括加速器、机场等信息，管理员发现会及时清除。
祝大家学的开心，Enjoy Coding~

**Comment by Ava-vivian** [comment]
支持！
非常好活动，爱来自瓷器（误）
对于0基础的编程小白来说太有用了，网站十分干净，也有很多引导。
对于编程的兴趣是最近才有的，以前完全没有接触过，最近那个GPT特别火，也有很多非常有意思的讨论，突然对人工智能产生了兴趣。这个假期甚至跟队学习参加了科大讯飞星火大模型的比赛。
（虽说实际上干的事情不过是收集数据（离数据挖掘还有亿段距离），做ppt，做宣传视频）
但是收获还是蛮大的，可以体验到通过代码，一步步实现各种功能，从而汇总成一个完整的项目，去满足一些目标需求的过程。在其中也见识到了技术上要学习的东西还有很多很多hhh
今后也要一步一步继续学习关于python乃至大模型的内容，希望这个活动之后也能越做越好！
（还有想找个学习搭子一起学，主要是能监督下进度，一起多讨论讨论，也能学得快些）
  ——from 张晋赫 23080114

**Comment by lkrr1213** [comment]
完整的完成了本次活动，超赞！
从准大一小白的角度体验后感觉活动难度设置合理，在过程中没有遇到什么问题，而且不枯燥
希望活动继续举办下去！！！

**Comment by Voskamp099** [comment]
我觉得这个活动真的特别特别好，希望如果可以的话继续办下去或者举办一些类似的活动。
说一下我的学习进度吧：我8月初开始的，python课学了一半多，crash course computer science学了十节课，指南中的visual studio和github账户均创建完成，读完了sjut生存指南（我承认挺慢的了😭，因为期间还在忙学英语出去玩等各项事宜，目前计划开学后抽时间跟着北大CS指南学习）。
对这个活动夸两点，第一，我感谢sjut指南，它启发我在日复一日重复的生活中思考，“我想要怎么过大学生活〞、“对于实现我的目标我需要做些什么、哪些时间是被无效浪费掉了的”。由此我的本科四年规划及对四年后的展望变得更清晰了。
第二，从python、Computer Science、C++到锦囊里汇总的各学校练习题，这些对我来说都是很有用很宝藏的资料，真的很感谢学长学姐们愿意这样分享出来，节省了我许多找资源的时间，也在努力时更有方向了。
总之，希望这类活动继续办下去，为我这样的新手小白们提供一些帮助和启发，再次感谢～！
